SU2_CFD/include/solvers/CFVMFlowSolverBase.inl

```diff
@@ -2533,7 +2550,12 @@ void CFVMFlowSolverBase<V, FlowRegime>::Friction_Forces(const CGeometry* geometr
       WallShearStress[iMarker][iVertex] = 0.0;
       for (iDim = 0; iDim < nDim; iDim++) {
         TauTangent[iDim] = TauElem[iDim] - TauNormal * UnitNormal[iDim];
-        CSkinFriction[iMarker](iVertex,iDim) = TauTangent[iDim] * factorFric;
+        /* --- in case of wall functions, we have computed the skin friction in the turbulence solver --- */
+        /* --- Note that in the wall model, we switch off the computation when the computed y+ < 5    --- */
+        /* --- We put YPlus to 1.0 so we have to compute skinfriction and the actual y+ in that case as well --- */
+        if (!wallfunctions || (wallfunctions && YPlus[iMarker][iVertex] < 5.0)) 
```

Comment:
This threshold should be defined somewhere central.

Reply:
Maybe together with other wall function parameters like number of iterations, tolerance, relaxation (even if they are not accessible via the config).

Reply:
yes, this is a threshold from the paper, but should be user-adjustable as well...